tools/deprecate_version.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR=$(realpath "$(dirname "$0")")
+
+mkdir -p build
+cd build
+
+# TODO(htuch): avoid creating virtualenv every time
+virtualenv deprecate_version
```

Comment:
nit: can we do a proper requirements.txt here for the venv?

Reply:
Sure. @danielhochman let me clean this up before diving into the Python minutiae, I'll ping you when it's ready.

Reply:
@danielhochman ready for review.